app/models/credence_game_response.rb

```diff
@@ -0,0 +1,70 @@
+class CredenceGameResponse < ActiveRecord::Base
+  belongs_to :credence_game
+  belongs_to :credence_question
+  belongs_to :first_answer, class_name: 'CredenceAnswer'
+  belongs_to :second_answer, class_name: 'CredenceAnswer'
+
+  def self.pick_random
+    enabled_gens = CredenceQuestion.where(enabled: true)
```

Comment:
This still says `gens`. Please rename it to the full name of what it actually is, such as `generations`, `genders`, `genomes`. I do not know what a `gens` is.